src/extension.ts

```diff
@@ -142,6 +143,12 @@ const toDiagnostic = (error: IValeErrorJSON): Diagnostic => {
     return diagnostic;
 };
 
+const readBinaryLocation = () => {
+    const configuration = workspace.getConfiguration();
+    const binaryPathRelativeToWorkspace =  configuration.get<string>('vscode-vale.path', 'vale');
+    return path.join(workspace.rootPath || '', binaryPathRelativeToWorkspace);
```

Comment:
I make the assumption here that the path should be relative to the workspace root folder.